features/login.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Login page
+  As an User
+  In order to access my account
+  I need to be able to login
+
+  Background:
+    Given I am on the "login" page
+    And the following users exist
+    | email                | password    |
+    | orphanage@random.com | my_password |
+
+  Scenario: Accessing the login page
+    Given I am on the "home" page
+    And I click "Login"
+    Then I should be on the "login" page
+
+  Scenario: Login User
+    And I fill in "Email" with "orphanage@random.com"
+    And I fill in "Password" with "my_password"
+    And I click "Log in"
+    Then I should be on the "home" page
```

Comment:
This one here would be more like `Then I should be redirected to the "Home" page`.

Clicking on 'Log in' doesn't take you to the home page, what happens instead is that your credentials are validated, and if all is well then you're redirected to the home page.